src/ProgressCircle/style.css

```
@@ -56,6 +56,15 @@
     display: block;
 }
 
+.ola_progressCircle-busy {
+    font: var(--font-caption);
+    color: var(--gray);
+    position: relative;
+    transition: opacity 200ms;
+    z-index: 1;
+    display: block;
+}
```

If the text is too wide it overflows the circle. So I suggest adding the following two properties to fix this error.
The padding is to prevent the text to exceed the circle border.

```suggestion
    padding: 0 var(--size-5);
    text-align: center;
}
```